test/box-luatest/box_info_memory_runtime_test.lua

```diff
@@ -0,0 +1,51 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_each(function()
+    g.server = server:new({alias = 'box_info_memory_runtime'})
+    g.server:start()
+end)
+
+g.after_each(function()
+    g.server:drop()
+end)
+
+local function runtime_memory()
+    return g.server:call('box.info.memory').runtime
+end
+
+local function allocate_tuples()
+    return g.server:eval([[do
+        local tuples = rawget(_G, 'tuples')
+        if tuples == nil then
+            tuples = {}
+            rawset(_G, 'tuples', tuples)
+        end
+        for i = 1, 100 do
+            table.insert(tuples, box.tuple.new({1, 2, 3}))
+        end
+    end]])
+end
+
+local function deallocate_tuples()
+    return g.server:eval([[do
+        rawset(_G, 'tuples', nil)
+        collectgarbage()
```

Comment:
Probably, it's better to call it twice.

https://stackoverflow.com/questions/28320213/why-do-we-need-to-call-luas-collectgarbage-twice

Reply:
I'm aware of this recipe, but I never saw a real need.